js-code-guidelines.md

```diff
@@ -129,6 +130,12 @@ Always run tests before pushing and PR'ing your code.
 
 ### Guidelines
 
+#### Supported Versions
+
+IPFS projects support both the Current and LTS versions of node.  Please see [nodejs.org](https://nodejs.org/) for what these currently are.
```

Comment:
Can we change that to "and **active** LTS versions". Given the current state of the IPFS ecosystem, I think we shouldn't put on the burden of supporting older versions.

I'd also write "Node.js" instead of "node".

Reply:
Done.